crates/runners/eigenlayer/src/bls.rs

```diff
@@ -53,8 +56,20 @@ async fn requires_registration_bls_impl(env: &GadgetConfiguration) -> Result<boo
     };
     let registry_coordinator_address = contract_addresses.registry_coordinator_address;
     let operator_state_retriever_address = contract_addresses.operator_state_retriever_address;
-    let operator = env.keystore()?.ecdsa_key()?;
-    let operator_address = operator.alloy_key()?.address();
+
+    let ecdsa_public = env
+        .keystore()
+        .iter_ecdsa()
+        .next()
+        .ok_or_else(|| Error::Keystore("No ECDSA keys found".into()))?;
```

Comment:
```suggestion
    let ecdsa_public = env
        .keystore()
        .first_local::<K256Ecdsa>()
        .ok_or_else(|| Error::Keystore("No ECDSA keys found".into()))?;
```

I added `first_local()` so it's easier to find all the cases of us grabbing the first key, for when key IDs are added.